packages/office-ui-fabric-react/src/components/Slider/Slider.types.ts

```diff
@@ -130,6 +128,25 @@ export interface ISliderProps extends React.ClassAttributes<SliderBase> {
    * @defaultvalue false
    */
   originFromZero?: boolean;
+
+  /**
+   * Optional flag to display tick marks on each step
+   */
+  showThumbTooltip?: boolean;
```

Comment:
Are we defaulting this prop to something?

Reply:
im not quite sure so far there has been no discussion regarding a default value for showThumbTooltip @aneeshack4 what r ur thoughts regarding this?

Reply:
We should keep it false and make it opt-in so that users who never expected the tooltip suddenly get one.

Reply:
ok cool makes sense just added default value of false